lua/hbac/command/init.lua

```diff
@@ -0,0 +1,49 @@
+local subcommands = require("hbac.command.subcommands")
+local telescope = require("hbac.telescope")
+
+local M = {
+	subcommands = {},
+}
+
+M.subcommands.close_unpinned = function()
+	subcommands.close_unpinned()
+	vim.notify("Closed unpinned buffers", "info", { title = "Hbac" })
+end
+
+M.subcommands.toggle_pin = function()
+	local bufnr, pinned_state = subcommands.toggle_pin()
+	vim.notify(bufnr .. " " .. pinned_state, "info", { title = "Hbac" })
+end
+
+M.subcommands.toggle_all = function()
+	local pin_all_state = subcommands.pin_all() and "pinned" or "unpinned"
+	vim.notify(pin_all_state .. " all buffers", "info", { title = "Hbac" })
+end
```

Comment:
Per the other conversation regarding `toggle_all`, should probably rename this
```suggestion
M.subcommands.toggle_all = function()
	local toggle_all_value = subcommands.pin_all() and "pinned" or "unpinned"
	vim.notify(toggle_all_value .. " all buffers", "info", { title = "Hbac" })
end
```

We can sleep on these though 😆  (these moments remind me of the Greek philosopher Cratylus who allegedly gave up language at the end of his life and spoke only with his finger, since he found words too slippery and incomplete)